package.json

```diff
@@ -38,18 +38,22 @@
     "!lib/**/*.map",
     "messages"
   ],
+  "resolutions": {
+    "@salesforce/core": "^8.15.1-dev.0"
+  },
```

Comment:
TODO: remove after QA is done
I had to add it b/c other deps were on a lower core version